Lesson2/src/Lesson2/Main.java

```diff
@@ -0,0 +1,173 @@
+// @author Shkarin Konstantin
+
+package Lesson2;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+public class Main {
+
+    public static void main(String[] args) {
+        // сортировка Bubble подсчет времени массива arr
+        MyArray arr = new MyArray(1000);
+        for (int i = 0; i < 1000; i++) {
+            int r = (int) (Math.random()*arr.getSize());
+            arr.add(r);
+        }
+        long start = System.nanoTime();
+        arr.sortBubble();
+        long finish = System.nanoTime();
+        long delta = finish-start;
+        TimeUnit.NANOSECONDS.toMillis(delta);
+        System.out.print("Bubble "+delta);
+        System.out.println(" ");
+
+        // сортировка Select подсчет времени массива arr1
+        MyArray arr1 = new MyArray(1000);
+        for (int i = 0; i < 1000; i++) {
+            int r = (int) (Math.random()*arr.getSize());
+            arr1.add(r);
+        }
+        start = System.nanoTime();
+        arr1.sortSelect();
+        finish = System.nanoTime();
+        delta = finish-start;
+        TimeUnit.NANOSECONDS.toMillis(delta);
+        System.out.print("Select "+ delta);
+        System.out.println(" ");
+
+        // сортировка Insert подсчет времени массива arr2
+        MyArray arr2 = new MyArray(1000);
+        for (int i = 0; i < 1000; i++) {
+            int r = (int) (Math.random()*arr.getSize());
+            arr2.add(r);
+        }
+        start = System.nanoTime();
+        arr.sortBubble();
```

Comment:
Видимо copy-paste ошибка, нужно было использовать
```suggestion
        arr2.sortInsert()
```